sigmadsp/helper/conversion.py

```diff
@@ -55,6 +55,34 @@ def float_to_frac_8_24(value: float) -> int:
     return int(value * 2**24)
```

Comment:
Seeing this, there should probably be an overflow check. Also, what happens for negative `value`s? Of course, these problems also affect the old implementation.

Reply:
the 8.24 / 5.23 values are signed (-128 to 128 / -16 to 16); not sure about overflow checks, I'd rather clamp the input to avoid surprises (wrap-around)